backend/src/main/java/ca/bc/gov/hlth/hnweb/controller/GroupMemberController.java

```diff
@@ -82,7 +83,7 @@
 			logger.info("addDependentResponse response: {} ", addDependentResponse);
 			
 			transactionComplete(transaction);
-			addAffectedParty(transaction, IdentifierType.PHN, addDependentResponse.getPhn());
+			addAffectedParty(transaction, IdentifierType.PHN, addDependentResponse.getPhn(), AffectedPartyDirection.INBOUND);
```

Comment:
Should it be INBOUND or OUTBOUND? What if dependent has already been added. There shouldn't be any entry in that scenario?

Reply:
All PHNs are logged for both Outbound and Inbound. The direction makes the entry unique. So the same PHN can logged for both directions if it's in the request and the response. This is logging the Inbound/response PHN from addDependentResponse.getPhn().